src/python/system/environment.py

```diff
@@ -768,6 +768,19 @@ def set_environment_parameters_from_file(file_path):
       set_value(environment_variable, environment_variable_value)
 
 
+def update_symbolizer_options(tool_options, symbolize_inline_frames=False):
+  """Checks and updates the necessary symbolizer options such as
+  `external_symbolizer_path` and `symbolize_inline_frames`."""
+  if 'external_symbolizer_path' not in tool_options:
```

Comment:
looks like this was a mistake, now this set up a wrong path for the symbolizer:

```
==1==WARNING: invalid path to external symbolizer!
==1==WARNING: Failed to use and restart external symbolizer!
```

Reply:
should be addressed with https://github.com/google/clusterfuzz/pull/1401, will check the logs